usb/esp_tinyusb/tinyusb.c

```diff
@@ -70,8 +71,14 @@ esp_err_t tinyusb_driver_install(const tinyusb_config_t *config)
 
     if (config->string_descriptor) {
         string_descriptor = config->string_descriptor;
+        if (config->string_descriptor_count != 0) {
+            string_descriptor_count = config->string_descriptor_count;
+        } else {
+            string_descriptor_count = 8; // Backward compatibility with esp_tinyusb v1.0.0
```

Comment:
This is deliberate. In case someone changes value of `USB_STRING_DESCRIPTOR_ARRAY_SIZE` in the future (to support more string descriptors) or removes it altogether (to support dynamic memory allocation for string descriptors), the backward compatibility with `esp_tinyusb v1.0.0` will not be broken.

Do you agree?

Reply:
Yes, sure.